cf-utils/cf-nat/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.eclipse.californium</groupId>
+		<artifactId>cf-bom</artifactId>
+		<version>2.3.0-SNAPSHOT</version>
+		<relativePath>../../bom</relativePath>
+	</parent>
```

Comment:
Is there a particular reason why you do not use *cf-utils* as the parent?

Reply:
My intention is to use the folder just as collection. If a pom is added to the "cf-utils" folder, that must be published as well in order to be able to us the cf-nat (see issue #1041 ).  Though the modules may vary much more than others "cf-nat" (independent) and (new to come) "cf-cli" (basic command line interface with picocli, highly dependent on californium), the pom on cf-utils will not contain too much information.